lectures/matplotlib.md

```diff
@@ -352,7 +352,54 @@ draw_graphs(style='dark_background')
 
 You can use the function to experiment with other styles in the list.
 
-If you are interested, you can even create [your own style sheets](https://matplotlib.org/stable/tutorials/introductory/customizing.html#defining-your-own-style).
+If you are interested, you can even create your own style sheets.
+
+Paraeters for your style sheets are stored in a dictionary-like variable `plt.rcParams`
+
+```{code-cell} python3
+---
+tags: [hide-output]
+---
+ 
+print(plt.rcParams.keys())
+
+```
+
+You can see there are many parameters you could set for your style sheets.
+
+You can change them by [changing the `matplotlibrc` file](https://www.datafantic.com/the-magic-of-matplotlib-stylesheets/).
+
+Or you can also update values stored in the dictionary-like variable `plt.rcParams`.
+
```

Comment:
Hi @HumphreyYang , to keep this brief, please cut the three lines above

That is, remove

"You can see..."
"You can change..."
"Or you can..."